src/build/build_helpers.jl

```diff
@@ -97,6 +96,14 @@ function get_jump_variable_for_sector(jm, sector)
     end
 end
 
+function get_jump_variable_for_param(jm, parameter::ParameterRef)
+    if parameter.subindex===nothing
+        return jm[parameter.model._parameters[parameter.index].name]
+    else
+        return jm[parameter.model._parameters[parameter.index].name][parameter.subindex]
+    end
+end
+
```

Comment:
aligning this with the structure of other variables; checking for the indexed version in this function rather than in swap_our_Ref_with_jump_var